cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -934,6 +935,13 @@
   private static final String TWO_STEP_PURGATORY_MAX_REQUESTS_DOC = "The maximum number of requests in two-step "
       + "(verification) purgatory.";
 
+  /**
+   * <code>request.reason.required</code>
+   */
+  public static final String REQUEST_REASON_REQUIRED_CONFIG = "request.reason.required";
+  private static final String REQUEST_REASON_REQUIRED_DOC = "Require specifying reason via " + REASON_PARAM + " parameter "
+      + "for non-dryrun POST request";
```

Comment:
Is this config controls this behavior for _all_  or _selected_ POST requests -- i.e. should `stop_proposal_execution` get a `reason` parameter, as well? If it is only for selected endpoints, can we enumerate them?